solver/llbsolver/file/backend.go

```diff
@@ -146,10 +146,12 @@ func rm(ctx context.Context, d string, action pb.FileActionRm) error {
 }
 
 func rmPath(root, src string, allowNotFound bool) error {
-	p, err := fs.RootPath(root, filepath.Join("/", src))
+	dir, base := filepath.Split(src)
```

Comment:
`Clean()` the `src` for safety before. And validate `base` not empty?

Reply:
Done